src/client/mapblock_mesh.cpp

```diff
@@ -1538,7 +1539,7 @@ void MapBlockMesh::consolidateTransparentBuffers()
 	for (const auto &t : m_transparent_triangles) {
 		if (current_buffer != t.buffer) {
 			if (current_buffer != nullptr) {
-				this->m_transparent_buffers.emplace_back(current_buffer, current_strain);
+				this->m_transparent_buffers.emplace_back(current_buffer, std::move(current_strain));
```

Comment:
if you move, the clear just after is NOP or unknown

Reply:
So what ? The variable is not used afterwards.

Reply:
I believe the C++ standard says that after a std::move, a value is in a valid but undefined state. It might not necessarily be empty. For example, it would be allowed for the implementation to copy instead of move. So to know for sure that it is empty, it has to be cleared.

In practice, almost no implementations behave like this. But the clear() will be optimized out anyway.

Reply:
Agree, .clear() guarantees that current_strain is empty after move.